src/ps.rs

```diff
@@ -228,6 +228,11 @@ impl ProcessTable {
 
 pub type ProcInfoTable = HashMap<Pid, ProcInfo>;
 
+// container_flags has at most one bit set.  If the state is unknown the value is zero.
+pub const CONT_ROOT: i8 = 1;
```

Comment:
This is not a bit vector, these are exclusive state values, so number 1, 2, 3 and fix the comment.